cranelift/isle/isle/src/sema.rs

```diff
@@ -1951,9 +1951,6 @@ impl TermEnv {
                 for def in defs {
                     // Check that the given variable name does not already exist.
                     let name = tyenv.intern_mut(&def.var);
-                    if bindings.vars.iter().any(|bv| bv.name == name) {
-                        tyenv.report_error(pos, format!("Variable '{}' already bound", def.var.0));
-                    }
```

Comment:
It's hilarious that this is what the PR boils down to :)

Reply:
Something about journeys vs. destinations etc etc ... but, yes, I'm happy it was a simple change!